packages/style/scss/lib/reset.scss

```diff
@@ -88,6 +88,7 @@ video {
     margin: 0;
     padding: 0;
     font: inherit;
+    line-height: 1.5 rem;
```

Comment:
It's actually based on the font-size so it would be `em` not `rem` but `line-height: 1.5;` implies the em

https://developer.mozilla.org/en-US/docs/Web/CSS/line-height